src/e2e-app/tabs/tabs-e2e.ts

```diff
@@ -0,0 +1,10 @@
+import {Component} from '@angular/core';
+import {MD_TABS_DIRECTIVES} from '@angular2-material/tabs/tabs';
+
+@Component({
+  moduleId: module.id,
+  selector: 'tabs-e2e',
+  templateUrl: 'tabs-e2e.html',
+  directives: [MD_TABS_DIRECTIVES]
+})
+export class TabsE2E {}
```

Comment:
Rather than calling this `TabsE2E`, I'd prefer to call it something like `BasicTabs`, since we can eventually have multiple scenarios for each component and also want to reuse these things for examples.